content/how-to/lazy-load/react.md

```diff
@@ -0,0 +1,151 @@
+---
+title: How to lazy load React 
+contributors:
+  - iammerrick
+---
+
+A component can lazily load dependencies without it's consumer knowing using higher order functions, or a consumer can lazily load it's children without its children knowing using a component that takes a function and collection of modules, or some combination of both.
```

Comment:
As someone very limited in using React, although I understand HOC, I think it would be awesome to have a click able link to something from @gaearon or React documentation.